api/src/identity-access-management/domain/services/agent-connect-oidc-authentication-service.js

```diff
@@ -0,0 +1,13 @@
+import { OidcAuthenticationService } from './oidc-authentication-service.js';
+
+const AGENT_CONNECT_CLAIM_MAPPING = {
+  firstName: ['given_name'],
+  lastName: ['usual_name'],
+  externalIdentityId: ['sub'],
+};
```

Comment:
Suggestion : 
Plutôt réutiliser la constante [DEFAULT_CLAIM_MAPPING](https://github.com/1024pix/pix/blob/dev/api/src/identity-access-management/domain/constants/oidc-identity-providers.js#L6) pour accentuer le fait qu'on ne surcharge que le mapping du lastName :

```suggestion
const AGENT_CONNECT_CLAIM_MAPPING = {
  ...DEFAULT_CLAIM_MAPPING,
  lastName: ['usual_name'],
};
```